frontend/apps/erd-web/app/erd/p/[...slug]/page.tsx

```diff
@@ -97,6 +97,7 @@ export default async function Page({
       />
     )
   }
+  console.warn(`Fetching content from ${contentUrl}`)
```

Comment:
Is there any reason for warn?

```suggestion
  console.info(`Fetching content from ${contentUrl}`)
```

Reply:
In our current vercel setting, warning+ log seems to be required.  If you change the threshold, it has not been verified how many logs will be generated using info

Reply:
I see. LGTM 👍🏻